conf/deployment/aws/ipi_3az_rhcos_3m_4w.yaml

```diff
@@ -0,0 +1,17 @@
+---
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'aws'
+  deployment_type: 'ipi'
+  region: 'us-east-2'
+  worker_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+  master_availability_zones:
+    - 'us-east-2a'
+    - 'us-east-2b'
+    - 'us-east-2c'
+  worker_replicas: 4
+  master_replicas: 3
+  worker_instance_type: 'm5.4xlarge'
```

Comment:
we should use i3 instead of  m5.4xlarge because we will get a direct nvme disk attached to an instance

Reply:
I can change this. Can you pls specify the full name.

Reply:
@iamniting i3en.2xlarge

Reply:
Done